docs/bigquery/usage.rst

```diff
@@ -10,6 +10,15 @@ BigQuery
 
 .. contents:: :local:
 
+Installation
+------------
+
+Install the ``google-cloud-bigquery`` library using ``pip``:
+
+.. code-block:: console
```

Comment:
I believe the language code you want here is `bash`.

Reply:
I copy-pasted from other installation instructions. Seems to render as intended with `console`.

![image](https://user-images.githubusercontent.com/247555/41170602-d3e9cf1c-6b01-11e8-91f0-4b423c17b14a.png)